src/components/views/Lobby.tsx

```diff
@@ -63,7 +64,6 @@ const LobbyPage = ({ stompWebSocketHook }) => {
                 if (lobbyData.owner.id === parseInt(localStorage.getItem("playerId"))) setOwnerMode(true);
             } catch (error) {
                 handleError(error, navigate);
-                alert("Unable to display lobby data");
                 localStorage.clear();
```

Comment:
If you are at it changing small things, can you change it so that it only deletes lobby and player related things and leaves the user token in the storage?